### DISCUSSION TITLE
Error:  no implicit conversion of Hash into Integer

### DISCUSSION THREAD

**Discussion by juliohm** [mmistakes/minimal-mistakes#3505] [General]
I have been a super happy user of minimal-mistakes for years and I would like to start by saying thank you!

Recently, my OS (Manjaro Linux) updated the Ruby version to 3.x and I can no longer build the github-page locally. Is there any workaround to build the page locally and force Ruby 2.x only in the folder of the project?

I am not familiar with basic concepts in web programming, so I appreciate any instruction on how to configure my desktop to keep it up to date and still be able to build my website with an old version of Ruby.

**Comment by mmistakes** [comment]
You're better off asking on the [Jekyll talk forums](https://talk.jekyllrb.com/) or searching on Stackoverflow. Your issue is a more general one that has to do with Ruby and Jekyll, and not this theme specifically.

You'll likely want to use a Ruby manager like [rbenv](https://github.com/rbenv/rbenv) that will allow you to install different versions of Ruby (and switch between them) easily.
